packages/usa-in-page-navigation/src/index.js

```diff
@@ -42,23 +44,35 @@ const setActive = (el) => {
 };
 
 /**
- * Return an array of all visible h2 and h3 headings from the designated main content region.
- * These will be added to the component link list.
+ * Return an array of visible headings from the designated content region.
+ * These items will be added to the component link list.
  *
- * @param {HTMLElement} mainContentSelector The designated main content region
+ * @param {HTMLElement} selectedContentRegion The content region the component should pull headers from
+ * @param {String} selectedHeadingTypes The list of heading types that should be included in the link list
  *
  * @return {Array} - An array of visible headings from the designated content region
  */
-const getSectionHeadings = (mainContentSelector) => {
-  const sectionHeadings = document.querySelectorAll(
-    `${mainContentSelector} h2, ${mainContentSelector} h3`
-  );
+const getSectionHeadings = (selectedContentRegion, selectedHeadingTypes) => {
+  const headingTypes = selectedHeadingTypes.indexOf(" ")
+    ? selectedHeadingTypes.split(" ")
+    : selectedHeadingTypes;
+  const contentHeadingTypes = [];
+
+  headingTypes.forEach((headingType) => {
+    if (!IN_PAGE_NAV_VALID_HEADINGS.includes(headingType)) {
+      throw new Error(
+        `Invalid heading type: ${headingType}. Please use one or more of the following: ${IN_PAGE_NAV_VALID_HEADINGS}`
```

Comment:
Should we include a note about not including punctuation in the error message?

Reply:
@mejiaj 
1. Yes, it means you need to define the attribute with one or more HTML heading elements (h1-h6). I added some clarity to the error message in https://github.com/uswds/uswds/pull/5444/commits/d7d27a661497cfd7741d9ab5e5e0ec56fb0e65b7
2. I added a note to the error message about not using punctuation in the attribute definition.